pkg/dockerregistry/server/registry.go

```diff
@@ -0,0 +1,20 @@
+package server
+
+import (
+	"github.com/docker/distribution"
+	"github.com/docker/distribution/context"
+	"github.com/docker/distribution/registry/middleware/registry"
+)
+
+// Registry represents a collection of repositories, addressable by name.
+// This variable holds the object created by the docker/distribution. We import
+// it into our namespace because there are no other ways to access it. In other
+// cases it is hidden from us.
+var Registry distribution.Namespace
```

Comment:
I don't like it, but apparently we have to live with it. Still make it lowercase, I don't want this be accessible from anywhere else.